controller/controller-common/src/main/java/com/pinterest/rocksplicator/controller/Cluster.java

```diff
@@ -0,0 +1,77 @@
+/*
+ *  Copyright 2017 Pinterest, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.pinterest.rocksplicator.controller;
+
+/**
+ * Storing cluster metadata (not including detailed segment/shard info)
+ *
+ * @author shu (shu@pinterest.com)
+ */
+public class Cluster {
+  public String namespace;
+  public String name;
```

Comment:
also can we add some comments on what is namespace and what is name in the cluster?